extension-requests/constants.js

```diff
@@ -32,3 +32,6 @@ const APPLY_FILTER_BUTTON = 'apply-filter-button';
 const CLEAR_BUTTON = 'clear-button';
 const SEARCH_ELEMENT = 'assignee-search';
 const LAST_ELEMENT_CONTAINER = '.virtual';
+const SORT_BUTTON = '.sort-button';
+const SORT_ASC_ICON = 'asc-sort-icon';
+const SORT_DESC_ICON = 'desc-sort-icon';
```

Comment:
- Can we not use this directly in `document.querySelector` or any other selector instead of making a variable? Do you have any reason for doing it this way?

Reply:
This is the convention followed throughout the repo where the selectors of common elements of the page are stored in constants.